### PR TITLE
Logs

### PR DESCRIPTION
**Pull request by uanabitarte**
Now we flush the database right away after the logs are persisted in the LoggerHandler. That way there is no need to call flush() after using the logger service.

### CODE REVIEW

File: src/Controller/DefaultController.php [hunk 1]
```diff
@@ -550,13 +545,13 @@ public function runRestoreJobBackupAction(Request $request, $idClient, $idJob, $
                   'sshArgs'    => $sshArgs
               ))
         );
-        $manager->persist($msg);
+       $manager->persist($msg);
+       $manager->flush();
```
Comment (elacunza) [inline src/Controller/DefaultController.php:549]: Bad indentation

File: src/Controller/DefaultController.php [hunk 2]
```diff
@@ -1248,7 +1236,6 @@ public function sortJobsAction(Request $request)
                 array(),
                 array('link' => $this->generateUrl('showClients'))
             );
-            $this->getDoctrine()->getManager()->flush();
             $this->get('session')->getFlashBag()->add(
```
Comment (elacunza) [inline src/Controller/DefaultController.php:1239]: We should flush something if we have reordered jobs? (I assume DB was updated, although I see no persist?

**Review by elacunza — CHANGES_REQUESTED** [review]

File: src/Controller/DefaultController.php (hunk 2 above)
Comment (uanabitarte) [reply src/Controller/DefaultController.php:1239]: I think there is no change in the DB When sorting jobs, as it only makes a query with an "orderBy" statement to sort the jobs by priority and show them in the form.

Comment (elacunza) [reply src/Controller/DefaultController.php:1238]: POST action is for saving the priority (order) of jobs. Must be saving to DB but can't see where really.

Comment (uanabitarte) [reply src/Controller/DefaultController.php:1238]: I have tested it, and the jobs are been reordered and the priority is saved in the DB correctly.

**Comment by elacunza** [comment]
Thanks Uxue, we'll leave that strange part for a separate issue

**Pull request merged**